src/dataset/pinecone_db_handler.py

```diff
@@ -17,10 +17,8 @@ def __init__(
     ):
         self.index_name = PINECONE_INDEX_NAME
         
-        PINECONE_API_KEY = os.getenv("PINECONE_API_KEY")
-        PINECONE_ENVIRONMENT = os.getenv("PINECONE_ENVIRONMENT")
-        assert PINECONE_API_KEY, "PINECONE_API_KEY environment variable not set."
-        assert PINECONE_ENVIRONMENT, "PINECONE_LOCATION environment variable not set."
+        PINECONE_API_KEY = os.environ["PINECONE_API_KEY"]
```

Comment:
is this change explicitly for error handling? Why not move them to `settings.py`?

Reply:
The `os.environ["PINECONE_API_KEY"]` instead of `os.getenv("PINECONE_ENVIRONMENT")` + assert was just a more concise version that did the ~same thing as before, where it calls an error if the env variable is not defined. And I just did, good idea.